code/style.css

```diff
@@ -1,7 +1,7 @@
 /* Global css variables used for colors */
 :root {
-  --primary: #a259ff;
-  --secondary: #b0a6ff;
+  --primary: #BDABBE;
+  --secondary: #32575D;
```

Comment:
I loved learning about how one can set key colors using --primary and --secondary - makes it so much easier to change the look in just a few seconds!